### PR TITLE
adding filter for approved templated

### PR DESCRIPTION
**Pull request by AkhileshNegi**
- Adding filter to show only approved templates when sending templated in flow

### CODE REVIEW

**Comment by pankaj-ag** [comment]
Let's write a test case for this task

**Comment by pankaj-ag** [comment]
Let's write test cases for that.

**Comment by pooja1425** [comment]
Seems like it's working.
Only **approved** templates are in the list.

File: test/glific_web/flows/flow_editor_controller_test.exs [hunk 1]
```diff
@@ -121,6 +152,47 @@ defmodule GlificWeb.Flows.FlowEditorControllerTest do
       assert Map.has_key?(response, "segments")
     end
 
+    test "templates2", %{conn: conn} = attrs do
```
Comment (pankaj-ag) [inline test/glific_web/flows/flow_editor_controller_test.exs]: ??

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

**Review by pankaj-ag — APPROVED** [review]

**Pull request merged**